Dockerfile.dev

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/openshiftio/fabric8-services-fabric8-cluster:c9398c
+FROM quay.io/openshiftio/fabric8-services-fabric8-cluster:bddfb4e
```

Comment:
Why not latest?

Reply:
Isn’t it what we want? To use the local build?

Reply:
Isn’t it what we want? To use the local build?

Reply:
for local build we'll be building the latest binary for cluster and just putting it in container using `COPY bin/cluster /usr/local/cluster/bin/cluster`.